### DISCUSSION TITLE
do we need bitmap.lock() ?

### DISCUSSION THREAD

**Discussion by sonavalepratham** [cefsharp/CefSharp#4822] [Q&A]
do we really need bitmap.Lock() in WritableBitmapRenderHandler.cs? as it's on dispatcher thread only. tried by removing Lock(), it's working fine for me.
https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Wpf/Rendering/WritableBitmapRenderHandler.cs#L179

**Comment by amaitland** [comment]
My understanding is that it's required to keep the WPF render thread synchronised correctly.

You can plugin your own render handler implementation if you want to run without locking.
